### PR TITLE
[AAE-11712] Validate models name while saving

### PR DESCRIPTION
**Pull request by graric**

### CODE REVIEW

**Comment by pmartinezga** [comment]
Take a look at the build, it looks like your changes are collapsing with other existing tests

**Review by pmartinezga — COMMENTED** [review]
It is surprising for me that we had tests covering the case of using underscore and using capital letters I need to investigate this before merging

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelingRestExceptionHandler.java [hunk 1]
```diff
@@ -87,6 +88,13 @@ public void handleModelNameConflictException(ModelNameConflictException ex,
         response.sendError(CONFLICT.value(), ex.getMessage());
     }
 
+    @ExceptionHandler(ModelNameInvalidException.class)
+    public void handleModelNameInvalidException(ModelNameInvalidException ex,
+                                                 HttpServletResponse response) throws IOException {
+        logger.error(ex.getMessage(), ex);
+        response.sendError(CONFLICT.value(), ex.getMessage());
+    }
```
Comment (pmartinezga) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelingRestExceptionHandler.java]: You are doing the same as the previous exception handler so I think it is better to merge the three we have for sending the conflict like this
```java
@ExceptionHandler({
    ModelNameConflictException.class,
    ModelNameInvalidException.class,
    ModelScopeIntegrityException.class
})
```

**Review by pmartinezga — APPROVED** [review]

**Review by pmartinezga — CHANGES_REQUESTED** [review]
asking request just because the mergify bot will merge this otherwise

**Review by pmartinezga — APPROVED** [review]

**Comment by graric** [comment]
Ref: https://github.com/Activiti/Activiti/issues/4273

**Pull request merged**